lib/ansible/cli/vault.py

```diff
@@ -76,6 +76,7 @@ def init_parser(self):
         create_parser = subparsers.add_parser('create', help='Create new vault encrypted file', parents=[vault_id, common])
         create_parser.set_defaults(func=self.execute_create)
         create_parser.add_argument('args', help='Filename', metavar='file_name', nargs='*')
+        create_parser.add_argument('-n', '--no-check', default=False, help='skip TTY check', dest='no_check', action='store_true')
```

Comment:
This argument name should be more descriptive, such as `--skip-tty-check`.  We also recommend dropping the shortopt of `-n` as that is overloaded and would lead to more confusion.